src/api.js

```diff
@@ -7,26 +7,30 @@ const gamesApi = axios.create({
 export const getReviews = (category) => {
   if (category) {
     return gamesApi.get(`/reviews/sort_by=${category}`).then(({ data }) => {
-        return data; 
-     
+      return data;
     });
   } else {
-    return gamesApi.get('/reviews').then(({ data }) => {
-        return data; 
-       
+    return gamesApi.get("/reviews").then(({ data }) => {
+      return data;
     });
   }
 };
 
 export const getReviewById = (review_id) => {
-  return gamesApi.get(`/reviews/${review_id}`).then(({ data: {review} }) => {
-      return review; 
-
-  })
-}
+  return gamesApi.get(`/reviews/${review_id}`).then(({ data: { review } }) => {
+    return review;
+  });
+};
 
 export const getCommentsByReviewId = (review) => {
   return gamesApi.get(`/reviews/${review}/comments`).then(({ data }) => {
-    return data; 
+    return data;
+  });
+};
+
+export const patchVoteByReviewId = (review, vote) => {
```

Comment:
Nice use of a function that deals with any vote increment handed to it (rather than separate functions to add and decrease the votes)